guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -5,6 +5,8 @@
 To configure an external DHCP server to use with {ProductName}, on a Red{nbsp}Hat Enterprise Linux server, you must install the ISC DHCP Service and Berkeley Internet Name Domain (BIND) packages.
 You must also share the DHCP configuration and lease files with {ProductName}. The example in this procedure uses the distributed Network File System (NFS) protocol to share the DHCP configuration and lease files.
 
+include::snip_firewalld.adoc[]
```

Comment:
This include is outside the .Procedure markup, while the others are inside it. I don't mind which one you pick, but they should be consistent.